.github/workflows/static-analysis.yaml

```diff
@@ -0,0 +1,17 @@
+name: "Static Analysis"
+on:
+  push:
+    branches: [master]
```

Comment:
This repo doesn't have a `master` branch. Is `master` the term for default branch?